### DISCUSSION TITLE
How much money or time would running the foundation take

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#22] [Q&A]
I think working on some estimate of cost or time would be useful for things like:
- legal fees to handle trademark
- technical and infrastructure feed to maintain DNS
- technical and infra fee to maintain other assets like secrets
- website creation and maintenance
- social media presence
- PR firm
- conference presence (needed?)
- finance (Assuming money flow in)

Things like that

Then there are additional work around crowdfunding and project based funding.

How will this money come?
How is it handled in case of "low income"?

**Comment by emmanuelbernard** [comment]
- fees for acquiring and maintaining trademark

**Reply by ebullient** [reply]
- Fees for trademarks: $5,800, assuming 2 marks (word mark and logo) with 2 classes registered in 2 jurisdictions. 
  - US: one mark ($1200), extra class ($350)
  - EU: one mark (€850), extra class (€50)
  - renewal is once every 5 years or 10 years.
- Domain registration: I already own them all and would donate. At renewal time (next year) we can decide how long to renew for and budget that recurring cost.
- Website creation and maintenance I do not consider something with cost: GH Pages works. I don't pay anyone anything to have a blog: this is no different. Domain assignment is then aligned with GH Pages. CloudFlare has an OSS sponsorship program that we can consider as traffic grows: https://blog.cloudflare.com/cloudflare-new-oss-sponsorships-program/
- I would like to defer a social media/PR firm unless/until we need it. I think we can get pretty far with grassroots + 6 degrees.
- I intend to work directly with conference organizers for initial bootstrap of presence based on colleagues that may be interested already being present at these events.
- I would like as much as possible to rely on generally available infrastructure. A GitHub application/bot is on my radar, but managing credentials is a concern for sure.
- Finance: See other post. This would be done with a fiscal host, with cost managed as money flows in so there are no recurring costs. They would then handle tax/accounting/etc.

In the early days, I would like to minimize/avoid recurring costs as much as possible. There are unavoidable initial costs (legal incorporation fees + bylaw review). We could wait on official trademark registration if we want to defer that cost.

**Comment by ebullient** [comment]
These are good questions, there is a semi/loose-draft budget, but I'm starting from the assumption of no (or very little money) rather than assuming gobs of it. Will close this for now (as the open ended question), and will leave it for soon-to-exist CFC to sort out.
